src/main/kotlin/com/autonomousapps/tasks/AdviceTask.kt

```diff
@@ -129,48 +136,53 @@ abstract class AdviceTask : DefaultTask() {
     val advicePrinter = AdvicePrinter(computedAdvice)
 
     var didGiveAdvice = false
+    val prettyReport = StringBuilder()
 
     if (!computedAdvice.filterRemove) {
       advicePrinter.getRemoveAdvice()?.let {
-        chatter.chat("Unused dependencies which should be removed:\n$it\n")
+        prettyReport.prettyAdvice("Unused dependencies which should be removed:\n$it\n")
         didGiveAdvice = true
       }
     }
 
     if (!computedAdvice.filterAdd) {
       advicePrinter.getAddAdvice()?.let {
-        chatter.chat("Transitively used dependencies that should be declared directly as indicated:\n$it\n")
+        prettyReport.prettyAdvice("Transitively used dependencies that should " +
+          "be declared directly as indicated:\n$it\n")
         didGiveAdvice = true
       }
     }
 
     if (!computedAdvice.filterChange) {
       advicePrinter.getChangeAdvice()?.let {
-        chatter.chat("Existing dependencies which should be modified to be as indicated:\n$it\n")
+        prettyReport.prettyAdvice("Existing dependencies which should be modified " +
+          "to be as indicated:\n$it\n")
         didGiveAdvice = true
       }
     }
 
     if (!computedAdvice.filterCompileOnly) {
       advicePrinter.getCompileOnlyAdvice()?.let {
-        chatter.chat("Dependencies which could be compile-only:\n$it\n")
+        prettyReport.prettyAdvice("Dependencies which could be compile-only:\n$it\n")
         didGiveAdvice = true
       }
     }
 
     // TODO add filter
     advicePrinter.getRemoveProcAdvice()?.let {
-      chatter.chat("Unused annotation processors that should be removed:\n$it\n")
+      prettyReport.prettyAdvice("Unused annotation processors that should be removed:\n$it\n")
       didGiveAdvice = true
     }
 
     if (didGiveAdvice) {
       chatter.chat("See machine-readable report at ${adviceFile.path}")
+      chatter.chat("See pretty report at ${advicePrettyFile.path}")
     } else {
-      chatter.chat("Looking good! No changes needed")
+      prettyReport.prettyAdvice("Looking good! No changes needed")
     }
 
     adviceFile.writeText(advices.toJson())
+    advicePrettyFile.writeText(prettyReport.toString())
```

Comment:
I think this will need a small change. It looks like you're writing out the console output to disk. This is ok and I have no problem with it. However, it breaks the current convention of using `-pretty.json`, which till now has only been used like this:
```
projectReportPrettyFile.writeText(unusedDirectDependencies.toPrettyString())
```
note the use of `toPrettyString()`, which is simply a json convention for printing with indentation and newlines, rather than just one long line. The output file for this also has the `.json` suffix, which isn't appropriate for plain english output like this.

If your goal is in fact to write the console output to disk, then I suggest three changes:
1. Change the filename extension to `.txt`
2. Change the outputDirectories method name to something like `getAdviceConsolePath()`
3. Change the file property name to `adviceConsole` and `adviceConsoleFile`.

I think this would be less surprising.